internal/utils/utils.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"context"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Used for Jitter in requeueing
+var DefaultRequeueDuration = (time.Minute * 5)
+
+const (
+	ErrorOp  = "error"
+	UpdateOp = "updated"
+	NoOp     = "unchanged"
+	CreateOp = "created"
+)
+
+type Controller struct {
```

Comment:
Why is this called Controller?

Reply:
It has been stolen from another operator and didn't change the name yet 😅